site-manage/site-manage-group-section-role-helper/tool/src/webapp/content/templates/GroupAutoCreate.html

```diff
@@ -21,7 +21,9 @@
                 </div>
             </div>
             <form rsf:id="groups-form">
-                <h3 rsf:id="prompt"></h3>
+				<div class="page-header">
```

Comment:
Indent inconsistency here.

Reply:
Are the indents in my files more consistent now?